config/slskd.example.yml

```diff
@@ -238,11 +238,27 @@
 #       on:
 #         - DownloadFileComplete
 #         - DownloadDirectoryComplete
-#       run: data/my_script.sh --json-to-process $EVENT
-#     my_logging_script:
+#       run:
+#         executable: ~ # if omitted, defaults to your operating system's $SHELL, or cmd.exe on Windows
+#         command: data/my_script.sh # the script reads the data from the $SLSKD_SCRIPT_DATA envar
+#     run_with_sh:
 #       on:
-#        - All
-#       run: data/log_slskd_events.sh $DATA
+#         - DownloadFileComplete
+#       run:
+#         executable: /bin/sh
+#         command: '-c "echo $SLSKD_SCRIPT_DATA >> downloads.txt"'
```

Comment:
this example is insecure: an attacker can create funny slskd_script_data which would allow them to execute any command on a shell (RCE).

Reply:
thanks, missed that